node-phantom.js

```diff
@@ -227,6 +227,11 @@ module.exports = {
         var prematureExitHandler = function (code, signal) {
           console.warn('phantom crash: code ' + code)
           server.close()
+
+          // required to signal up the call stack that phantom has crashed
+          // the catching block will signal a error return code allowing
+          // a supervisor process to restart
+          throw("premature phantom crash");
```

Comment:
So this wouldn't work if you just did process.exit(2) or some non zero return code?  Seems strange that it needs an explicit exception from JS for some monitoring process to catch it.

If this gets you unstuck though feel free to land.

Please fixup lint here, I think there should be no semicolon and it's not a function so I think just have a space instead of (...).